README.md

```diff
@@ -2,3 +2,20 @@
 
 A buildpack for including the ICU library into a Heroku dyno. This is required
 as a dependency for several tools.
+
+## How it works
+
+This buildpack makes ICU 4.8 available in the `/app/icu` directory for both
+compilation and runtime uses. The primary purpose of this is to allow Charlock
```

Comment:
What exactly do you mean by compilation vs runtime here? What's the difference, and why do we need to account for both?